content/en/community/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Welcome to the community"
```

Comment:
I love this page and I think we can reformat it nicely with [Hextra cards](https://imfing.github.io/hextra/docs/guide/shortcodes/) when time comes!